src/models/request.js

```diff
@@ -11,8 +9,5 @@ export default (sequelize, DataTypes) => {
     Request.belongsTo(models.Users, { foreignKey: 'requesterId', targetKey: 'id' });
     Request.belongsTo(models.Trip, { foreignKey: 'tripId', targetKey: 'tripId' });
   };
-  Request.afterUpdate((data) => {
-    emitter.emit('request-updated', data);
```

Comment:
Why is the emitter removed from the afterUpdate hook?

Reply:
Because we are updating different things in requests, and the story to emit notification was when approved or rejected, I talked with victor and we moved it in [ request controller](https://github.com/andela/the_spinners-backend/blob/987b0965697ed7f3453ad0ac36ef834e0d408c5e/src/controllers/request.controller.js#L31) where the action is taken so that notifications won't be sent unnecessarily.